rune/go.mod

```diff
@@ -3,27 +3,35 @@ module github.com/inclavare-containers/rune
 go 1.14
 
 require (
+	github.com/alibaba/inclavare-containers/epm v0.0.0-20201022145650-08a6ceb72088
+	github.com/alibaba/inclavare-containers/rune v0.0.0-00010101000000-000000000000
 	github.com/checkpoint-restore/go-criu v0.0.0-20191125063657-fcdcd07065c5
-	github.com/cilium/ebpf v0.0.0-20200319110858-a7172c01168f // indirect
 	github.com/containerd/console v1.0.0
 	github.com/coreos/go-systemd/v22 v22.0.0
 	github.com/cyphar/filepath-securejoin v0.2.2
 	github.com/docker/go-units v0.4.0
+	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-restruct/restruct v0.0.0-20191227155143-5734170a48a1
-	github.com/godbus/dbus/v5 v5.0.3 // indirect
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.2
 	github.com/moby/sys/mountinfo v0.1.3
 	github.com/mrunalp/fileutils v0.0.0-20171103030105-7d4729fb3618
-	github.com/opencontainers/runc v0.0.0-20200429081546-85c44b190e42
+	github.com/opencontainers/runc v0.0.0-20200429033603-85c44b190e42
```

Comment:
I think EPM and bundle cache don't depend on `github.com/opencontainers/runc` directly. EPM only depends on `rune`
and `rune` depends on `runc`. If you require `github.com/inclavare-containers/rune`(rune declares itslef) rather than `github.com/alibaba/inclavare-containers/rune`, you needn't to require `github.com/opencontainers/runc` any more.

Reply:
By the way, the invalid runc timestamp is fixed by this commit: https://github.com/alibaba/inclavare-containers/pull/223